rts/Rendering/Map/InfoTexture/Modern/Path.cpp

```diff
@@ -208,14 +208,18 @@ void CPathTexture::Update()
 	}
 
 	// spread update across time
-	constexpr int TEX_SIZE_TO_UPDATE_EACH_FRAME = 128*128;
+	constexpr int TEX_SIZE_TO_UPDATE_EACH_FRAME = 256*256;
 	if (updateProcess >= texSize.y) updateProcess = 0;
 
-	int start = updateProcess;
-	const int updateLines = std::max(TEX_SIZE_TO_UPDATE_EACH_FRAME / texSize.x, ThreadPool::GetNumThreads());
+	const int start = updateProcess;
+	const int hst = start >> 1;
+	const int updateLines = std::max(TEX_SIZE_TO_UPDATE_EACH_FRAME / texSize.x, ThreadPool::GetNumThreads()) & 0xFFFE;
```

Comment:
`0xFFFE` looks magic, could probably use a comment on what it does and why
```suggestion
	constexpr int EVEN_NUMBERS_ONLY = 0xFFFE; // even numbers are needed because blablabla
	const int updateLines = std::max(TEX_SIZE_TO_UPDATE_EACH_FRAME / texSize.x, ThreadPool::GetNumThreads()) & EVEN_NUMBERS_ONLY;
```